app/src/main/java/org/bspb/smartbirds/pro/ui/fragment/MonitoringCommonFormFragment.java

```diff
@@ -206,7 +206,7 @@ public void onLoadFinished(Loader<Cursor> loader, Cursor cursor) {
             if (!manualLocation) {
                 updatingLocationFromGPS = true;
                 try {
-                    locationView.setText(location.name.bg);
+                    locationView.setText(location.name.get(Locale.getDefault()));
```

Comment:
isn't this supposed to use `R.strings.locale`. I think that would be updated dynamically when language is changed.